### PR TITLE
Feature/file structure

### PR DESCRIPTION
**Pull request by hayleyw7**
# Description drawing

- Added and sorted component files
- Installed Cypress
- Deleted default files that weren't needed
- Works on #3 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [x] local host 3000
- [ ] console.log()
- [ ] dev tools
- [ ] cypress testing

# Checklist:

- [x] My code follows the style guidelines of this project/linted
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,5 +47,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "cypress": "^8.3.1"
   }
 }
```
Comment (Matt-Roden) [inline package.json:54]: Cypress!!

File: public/index.html [hunk 2]
```diff
@@ -7,34 +7,11 @@
...
-      If you open it directly in the browser, you will see an empty page.
-
-      You can add webfonts, meta tags, or analytics to this file.
-      The build step will place the bundled scripts into the <body> tag.
-
-      To begin the development, run `npm start` or `yarn start`.
-      To create a production bundle, use `npm run build` or `yarn build`.
-    -->
   </body>
 </html>
```
Comment (Matt-Roden) [inline public/index.html:17]: Nice, I like starting with a more blank slate. I think it will help our learning

**Review by Matt-Roden — APPROVED** [review]
🥳 Looks good. I think we have a nice modular file structure to start with! Also a nice clean state to start with :)

File: public/index.html (hunk 2 above)
Comment (joanafbrito) [reply public/index.html:17]: ^^

File: src/components/App/App.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react';
-import { getPokedexData } from './apiCalls'
+import { getPokedexData } from '../../apiCalls'
```
Comment (joanafbrito) [inline src/components/App/App.js:3]: Great job with changing the path

**Review by joanafbrito — APPROVED** [review]
Thank you for structuring those files, and great job with the file Path

**Pull request merged**